napari/viewer.py

```diff
@@ -30,10 +31,16 @@ class Viewer(ViewerModel):
         Dimension names. by default they are labeled with sequential numbers
     show : bool, optional
         Whether to show the viewer after instantiation. by default True.
+    console_local_update : bool, optional
+        By default try to capture the locals from the lowest caller outside
+        of napari, and inject them into the qtconsole when created.
+        This may create object leaks.
```

Comment:
Can you elaborate on this point? Do you mean that users interacting with the console might find that objects are hanging around longer than expected? I would expect to have to `del obj` inside the console if I do this, but is there anything worse than that that you think could happen?